scripts/build.sh

```diff
@@ -13,35 +13,67 @@ set -e
 CURR_VER_FILE="/tmp/current-server-version"
 
 if [ -z "$VERSION" ] || [ "$VERSION" == "latest" ]; then
-    VERSION=$(sh "$(dirname "$0")"/get-latest-version.sh)
+    VERSION=$(bash "$(dirname "$0")"/get-latest-version.sh)
```

Comment:
Is there a particular reason for this to be changed from `sh`` to `bash`? Currently, this should not be an issue, becausse the base images are based on debian, but when a minebase image is used in the future which is based on alpine, for example, this will fail at runtime. 

Maybe we could add a check for available shells which have common compatibilities in the image and pass it in via a variable. But I would not see this as necessary by now, just maybe something worth adding later on to avoid problems.